content/en/docs/reference/glossary/endpoint.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Endpoints
+id: endpoints
+date: 2020-04-23
+full_link: 
+short_description: >
+  Endpoints tracks the IP Addresses of the objects the service send traffic to.
```

Comment:
@nevermyuk, to me this does not read like idiomatic English.

Reply:
It seems like having Too many plural words!

Reply:
Maybe "Endpoints track the IP addresses of the resources targeted by the associated Service."

Reply:
```suggestion
  Endpoints tracks the IP addresses of the objects that a Service sends traffic to.
```